src/main/java/ua/tc/marketplace/repository/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package ua.tc.marketplace.repository;
+
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import ua.tc.marketplace.model.entity.User;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+  Optional<User> findByEmail(String username);
+  Optional<User> findById(Long id);
```

Comment:
this method seams like redundant. It is automatically provided by Spring and we do not need to define it explicitly.